src/dolphin/_show_versions.py

```diff
@@ -0,0 +1,104 @@
+"""Utility methods to print system info for debugging.
+
+Adapted from `rasterio.show_versions`,
+which was adapted from `sklearn.utils._show_versions`
+which was adapted from `pandas.show_versions`
+"""
+import importlib
+import platform
+import sys
+
+__all__ = ["show_versions"]
+
+
+def _get_sys_info():
+    """System information.
+
+    Returns
+    -------
+    dict:
+        system and Python version information
+    """
+    blob = [
+        ("python", sys.version.replace("\n", " ")),
+        ("executable", sys.executable),
+        ("machine", platform.platform()),
+    ]
+
+    return dict(blob)
+
+
+def _get_opera_info():
+    """Information on system on core modules.
+
+    Returns
+    -------
+    dict:
+        dolphin / opera module information
+    """
+    # pylint: disable=import-outside-toplevel
+    import dolphin
```

Comment:
Why are we hiding the `dolphin` import here instead of putting it at the top level like `pylint` wants?